### ISSUE TITLE
Parameters for reproducing results

### ISSUE THREAD

**Issue by ncherel** [fizyr/keras-retinanet#485]
Hi,
First of all, congratulations on the work done.

I'm trying to reproduce the results you got on COCO with Resnet50 (in the README).
Is there any parameters to set (bach size, learning rate, image size, whatever...) or `keras-retinanet coco /path/to/coco` will do the job ?

Thanks

**Comment by hgaiser** [comment]
The current model for COCO is a concatenation of many training sessions. Not because that gave the best result, but because it was more convenient. As a result, I don't know for how many epochs you should train to expect the same output.

The learning rate and image sizes are the same as is used when unspecified.
